doc/base/classes.xml

```diff
@@ -530,7 +534,11 @@
 			<argument index="0" name="path" type="String">
 			</argument>
 			<description>
-				Load a resource from the filesystem, pass a valid path as argument.
+				Load a resource from the filesystem located at 'path'. Note: resource paths can be obtained by right clicking on a resource in the Assets Pannel and choosing "Copy Path".
+				[codeblock]
+				# load a scene called main located in the root of the project directory
+				var main = load("res://main.tscn")
+				[/codeblock]
 			</description>
 		</method>
 		<method name="log">
```

Comment:
I think we should rename this method to logn.

Reply:
Or maybe `ln`?

Reply:
I thought about "ln" just didn't want to confuse people with length.